odoo/addons/test_translation_import/tests/test_term_count.py

```diff
@@ -341,3 +342,20 @@ def test_export_import_csv(self):
         })
         with mute_logger('odoo.addons.base.models.res_lang'):
             import_fr.with_context(create_empty_translation=True).import_lang()
+
+
+@tagged('post_install', '-at_install')
```

Comment:
Is it post_install to avoid serialising with other at_install tests?

Reply:
In its actual form, it considers *all* modules.  So it could be run at any time.
But we could restrict it to all installed modules, in which case `post_install` is the way to go.
I don't know which is best.